action.yml

```diff
@@ -19,7 +25,7 @@ runs:
   steps:
     - run: |
         yarn --cwd ${{ github.action_path }} --production=true
-        yarn --cwd ${{ github.action_path }} start --issue "${{ inputs.issueUrl }}" --file "${{ github.action_path }}/results.json"
+        yarn --cwd ${{ github.action_path }} start --issue "${{ inputs.issueUrl }}" --file "${{ github.action_path }}/results.json" --evmNetworkId "${{ inputs.evmNetworkId }}" --token "${{ inputs.token }}" --settings ${{ inputs.settings }}
```

Comment:
I believe it'd be better to have a separate function for workflows like in [assistive-pricing](https://github.com/ubiquibot/assistive-pricing/blob/development/src/index.ts) instead of using the CLI, also `evmNetworkId` is part of the settings